GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -420,6 +425,7 @@ public void OnFileConvertedToFull(string relativePath)
 
         public virtual void OnFileRenamed(string oldRelativePath, string newRelativePath)
         {
+            this.newlyCreatedFileAndFolderPaths.Add(newRelativePath);

```

Comment:
Do we actually need this? How often does someone *rename* into a temp path? If we don't expect benefit from this, I would remove it so that we're not adding unnecessary contention on the concurrent data structure.

Reply:
I will remove this for now because we aren't measuring how often this is happening so we can't measure any benefit if we don't know what or if there are current difficulties that are from this.